.sass-lint.yml

```diff
@@ -11,7 +11,7 @@ rules:
   no-url-protocols: 0
   class-name-format:
     - 1
-    - convention: ^([a-z0-9]+([A-Z][a-z0-9]+)*)(-{1}([a-z0-9]+([A-Z][a-z0-9]+)*)){0,2}$
+    - convention: ^([a-z0-9]+([A-Z][a-z0-9]+)*)(-{1}([a-z0-9]+([A-Z][a-z0-9]+)*)){0,4}$
     - convention-explanation: 'Classes should be camelCase and separated by no more than 5 hyphens'
```

Comment:
Does this need to be 4, or will 3 suffice?  I don't see any rules below that warrant support for 4 hyphens. 

Additionally, the explanation should match whatever value we select. 

Also, we probably should have a team discussion about this rule in general.  If we're having to keep changing things because of unavoidable third-party dependency, then maybe we should discuss the merits of this rule in general, because we may be defeating the purpose of this rule by setting a threshold so high to begin with.